components/centraldashboard/public/components/main-page.pug

```diff
@@ -64,6 +66,7 @@ app-drawer-layout.flex(narrow='{{narrowMode}}',
             section.information
     app-header-layout(fullbleed)
         app-header(slot='header', hides, hidden$='[[notFoundInIframe]]')
+            header-banner(msg='[[bannerMessages]]', language='[[language]]')
```

Comment:
I think something is off with the templating somewhere

Testing on dev
![image](https://github.com/user-attachments/assets/32994715-8c1c-4093-8bac-c39db3fcf6d7)

I see that the banners are pushing down the namespace selector, but its not pushing down the page content. So the header is going over items in the dashboard page.

Reply:
Oh great, this is inconsistent...
![image](https://github.com/user-attachments/assets/c47d1e05-0d2c-4c6c-b71a-31975e1d3a1e)

![image](https://github.com/user-attachments/assets/a1241377-e945-48ad-b598-d0341f5232ff)

Only a few refreshes in between

Reply:
I have possibly fixed this... or not. I am not quite sure if the code I change works always, or just got lucky with my 300 refreshes.

Reply:
seems fixed 
![image](https://github.com/user-attachments/assets/1d95d530-ee49-4764-b70b-73f7595a336f)